recipe/meta.yaml

```diff
@@ -98,18 +90,15 @@ test:
     ] %}
     {% for each_hdf5_lib in hdf5_libs %}
     - test -f $PREFIX/lib/lib{{ each_hdf5_lib }}.a                           # [unix]
-    - test -f $PREFIX/lib/lib{{ each_hdf5_lib }}.dylib                       # [osx]
-    - test -f $PREFIX/lib/lib{{ each_hdf5_lib }}.so                          # [linux]
+    - test -f $PREFIX/lib/lib{{ each_hdf5_lib }}${SHLIB_EXT}                 # [unix]
     - if not exist %PREFIX%\\Library\\lib\\{{ each_hdf5_lib }}.lib exit 1    # [win]
     - if not exist %PREFIX%\\Library\\bin\\{{ each_hdf5_lib }}.dll exit 1    # [win]
-    {% endfor %}
-
     # Inspect linkages of HDF5.
-    {% for each_hdf5_lib in hdf5_libs %}
     - otool -L $PREFIX/lib/lib{{ each_hdf5_lib }}.dylib                      # [osx]
     {% endfor %}
-    - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
-    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
+
+    - conda inspect linkages -p $PREFIX $PKG_NAME  # [unix]
+    - conda inspect objects -p $PREFIX $PKG_NAME   # [osx]
```

Comment:
These can be dropped. `conda-build` 3 already outputs this info and these commands are deprecated.

Reply:
👍